src/llm/apis/openai_completions.cpp

```diff
@@ -259,8 +259,6 @@ absl::Status OpenAIChatCompletionsHandler::parseMessages() {
                     if (member->value.IsString()) {
                         request.chatHistory.back().insert({member->name.GetString(), member->value.GetString()});
                     }
-                } else {
-                    return absl::InvalidArgumentError("Invalid message structure - content should be string or array");
```

Comment:
TODO: removal of this else requires multiple unit tests, removal exposes new paths for tests